packages/shortest/src/core/runner/test-reporter.ts

```diff
@@ -1,112 +1,126 @@
 import pc from "picocolors";
-import { AssertionError, TestFunction } from "../../types";
-
-export type TestStatus = "pending" | "running" | "passed" | "failed";
-
-interface TokenMetrics {
-  input: number;
-  output: number;
-}
-
-interface TestResult {
-  name: string;
-  status: TestStatus;
-  error?: Error;
-  tokenUsage?: TokenMetrics;
-}
+import { FileResult, TestResult, TestStatus } from "@/core/runner/index";
+import { AssertionError, TestFunction } from "@/types/test";
 
 export class TestReporter {
   private currentFile: string = "";
   private testResults: Record<string, TestResult> = {};
   private startTime: number = Date.now();
   private currentTest: TestResult | null = null;
+  private legacyOutputEnabled: boolean;
+  // private reporterLog: Log;
```

Comment:
This and related code is coming from https://github.com/anti-work/shortest/pull/320. I kept it here to make the merging in that PR easier.